DataFormats/TCDS/interface/TCDSRaw.h

```diff
@@ -0,0 +1,130 @@
+#ifndef DATAFORMATS_TCDS_TCDSRAW_H
+#define DATAFORMATS_TCDS_TCDSRAW_H
+
+//---------------------------------------------------------------------------
+//!  \class TCDSRaw
+//!  \brief Structure of raw data from TCDS FED 1014
+//!
+//!  \author Remi Mommsen - Fermilab
+//---------------------------------------------------------------------------
+
+#include <cstdint>
+#include "EventFilter/FEDInterface/interface/fed_header.h"
+#include "EventFilter/FEDInterface/interface/fed_trailer.h"
+
+#pragma pack(push)
+#pragma pack(1)
+
+namespace tcds {
+
+  struct Sizes_v1
+  {
+    const uint8_t headerSize;
+    const uint8_t summarySize;
+    const uint8_t L1AhistSize;
+    const uint8_t BSTSize;
+    const uint8_t reserved0;
+    const uint8_t reserved1;
+    const uint8_t reserved2;
+    const uint8_t BGOSize;
+  };
+
+  struct Header_v1
+  {
+    const uint64_t macAddress;
+    const uint32_t swVersion;
+    const uint32_t fwVersion;
+    const uint32_t reserved0;
+    const uint32_t recordVersion;
+    const uint32_t runNumber;
+    const uint32_t reserved1;
+    const uint32_t activePartitions2;
+    const uint32_t bstReceptionStatus;
+    const uint32_t activePartitions0;
+    const uint32_t activePartitions1;
+    const uint32_t nibble;
+    const uint32_t lumiSection;
+    const uint16_t nibblesPerLumiSection;
+    const uint16_t triggerTypeFlags;
+    const uint16_t reserved5;
+    const uint16_t inputs;
+    const uint16_t bxid;
```

Comment:
This was called "bcid" in the previous struct of FED1024: just mentioning here in case it was a typo (but I don't think so, because I don't see any possible ambiguity with the corresponding "bxid" of the struct L1aInfo_v1)

Reply:
The change was intentional. Both acronyms are commonly used for the bunch crossing identifier. I chose to use bxid everywhere to be consistent.